setup.py

```diff
@@ -14,21 +14,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import distutils.core as mod_distutilscore
+from setuptools import setup
 
-mod_distutilscore.setup(
+
+with open('README.md') as f:
+    long_description = f.read()
+
+setup(
     name='gpxpy',
     version='1.3.3',
     description='GPX file parser and GPS track manipulation library',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
     license='Apache License, Version 2.0',
     author='Tomo Krajina',
     author_email='tkrajina@gmail.com',
     url='http://www.trackprofiler.com/gpxpy/index.html',
     packages=['gpxpy', ],
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
     classifiers=[
         "Programming Language :: Python",
         "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
```

Comment:
Could also add 3.7 here; tests all seem fine there.

Reply:
👍 

Added in https://github.com/tkrajina/gpxpy/pull/147.